coalib/bearlib/aspects/Smell.py

```diff
@@ -13,13 +13,17 @@ class Smell:
     """
     class docs:
         example = """
-        * Feature envy
-        * Data clump
-        * Too large class
-        * Too long parameter list
-        etc...
+        =begin
+        Example of Ruby code with data clumps and methods with too many
+        parameters.
+        =end
+        class Dummy
+            def x(y1,y2, y3, y4, y5, y6, y7, y8, a); end
```

Comment:
this example (especially `y1,y2`) suggests that whitespace problems are part of `Smell`.